### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by HarshithBorundia** [community/community#59004] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Feature Area

Projects

### Body

I made a project. My personal website .I got free domain through student developer pack. Everything works fine. Today I updated my index.html file in main branch.When I try to load my website it shows 404 error.Why?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by Relluem94** [comment]
Do you have a Apache or Nginx running?
check file names and paths, also make sure permission are right.

**Reply by HarshithBorundia** [reply]
No. Can you help me out.

**Reply by Relluem94** [reply]
sure, but i need more information. How did you host the page? Is it on a linux or a windows? cloud?

**Reply by HarshithBorundia** [reply]
I used mailcheap.Got a domain and hosted it.

**Reply by Relluem94** [reply]
can you confirm that you can host websites there? unfortunately i only found an email hoster with this name. 
also i do not find anything about hosting a webpage (php or html) in there documentation: https://www.mailcheap.co/help/nms/

i recommend using a proper hoster.

do you host it with github pages? if yes what did you configure?

can i help you on anything else or is all good?

**Reply by Relluem94** [reply]
As i found your Github Page (https://harshithborundia.github.io/Personal-Website/#contact)

Take a look at the Documentation for hosting github pages on a custom domain:
https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site

**Reply by Relluem94** [reply]
@HarshithBorundia as your site now works, (https://harshithborundia.me/) can you close that Question?

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @HarshithBorundia ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
